HomeWorkSolution/Task2/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {   
+            int num1 = InputNumber();
+            int num2 = InputNumber();
+            Console.WriteLine("Enter a math sign");
+            char sign = Console.ReadLine()[0]; //Read 1st symbol of an entered string
+
+            Console.WriteLine("Try to calculate it yourself!");
+            Console.WriteLine("Your answer is:");
+            int result = int.Parse(Console.ReadLine());          
+            if (result == WorkWithTwoNumbers(num1, num2, sign))
+                Console.WriteLine("Done! You're goddamn right. Totally sure you can type binary code instead of c#. Think about it! :)");
+            else if(result > WorkWithTwoNumbers(num1, num2, sign))
+                Console.WriteLine("Less than you think! :(");
+            else
+                Console.WriteLine("More than you think! :(");
+            Console.ReadLine();
+        }
+        /// <summary>
+        /// Return a number entered in Console
+        /// </summary>
+        /// <returns></returns>
```

Comment:
returns section also should be documented :). anyway - great job!

Reply:
Fixed